MetaMask/metamask-extension
# Part of #17670 for: accreditation.component.js

## Explanation
This PR is a part of the issue #17670 Replace deprecated Typography with Text component for file 
**ui\components\app\transaction-decoding\components\ui\accreditation\accreditation.component.js**


## Screenshots/Screencaps


### Before
<img width="1440" alt="Screenshot 2023-07-24 at 12 09 58 PM" src="https://github.com/MetaMask/metamask-extension/assets/8112138/f217b266-c859-4e8a-9b73-8323657d4274">


### After

<img width="810" alt="Screenshot 2023-07-24 at 12 10 21 PM" src="https://github.com/MetaMask/metamask-extension/assets/8112138/52ac22bc-bdc6-44ea-9894-9d4a8ad774dc">


https://github.com/MetaMask/metamask-extension/assets/8112138/04a9fd2b-6a4a-4cc1-989e-a55cf2e09911



## Manual Testing Steps
- Pull this branch 
- Check storybook
- Go over the testing steps in this PR https://github.com/MetaMask/metamask-extension/pull/12881

## Pre-merge author checklist


## Pre-merge reviewer checklist


If further QA is required (e.g. new feature, complex testing steps, large refactor), add the `Extension QA Board` label.

In this case, a QA Engineer approval will be be required.
